verity/src/helm/templates/secret-provider.yaml

```diff
@@ -8,4 +8,7 @@ spec:
     objects: |
         - objectName: "{{ .Values.secrets.awsSecretsNameCredentialsTf }}"
           objectType: "secretsmanager"
-          objectAlias: "credentials-tf.json"
\ No newline at end of file
+          objectAlias: "credentials-tf.json"
+        - objectName: "{{ .Values.secrets.awsSecretsNameCredentials }}"
```

Comment:
once the branch is updated with the latest `main` changes makes sense to declare this value in `values.yaml` in the same way as `Values.secrets.awsSecretsNameCredentialsTf`

Reply:
done